app/assets/javascripts/components/description/DescriptionContent.js

```diff
@@ -24,12 +25,15 @@ const replaceStoryLink = (domNode, linkedStories) => {
 }
 
 const DescriptionContent = ({ description, isReadonly, linkedStories, onClick }) => {
-  const isEmpty = (!description || !description.length);
-  description = Parser(description, { replace: domNode =>
+  const converter = new showdown.Converter();
+  const descriptionHTML = converter.makeHtml(description);
```

Comment:
Don't we have this https://github.com/Codeminer42/cm42-central/blob/123afbc98b539b4e1c9204ef0839ad902d9964ed/app/assets/javascripts/libs/globals.js#L6 for that already?

Reply:
Hmmm... I didn't see that. I'll try it now!